src/lib/rest/rest.cpp

```diff
@@ -811,6 +811,18 @@ void firstServicePath(const char* servicePath, char* servicePath0, int servicePa
 
 
 
+/* ****************************************************************************
+*
+* isOriginAllowedForCORS - checks the Origin header of the request and returns
+* true if that Origin is allowed to make a CORS request
+*/
+bool isOriginAllowedForCORS(std::string requestOrigin)
+{
+  return ( (requestOrigin != "") && ((strcmp(corsOrigin, "__ALL") == 0) || (strcmp(requestOrigin.c_str(), corsOrigin) == 0)) );
```

Comment:
If think it should change: `return ( ( .. ) );` -> `return ( )` per [M11 style rule](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/contribution_guidelines.md#m11-spaces)

> Also, after `(` and before `)`, spaces MUST NOT be present.

Reply:
Fixed in da358b6